README.md

```diff
@@ -1,2 +1,70 @@
-# Math-Magicians
-added later
+# Getting Started with Create React App
```

Comment:
Please add a descriptive project title and description in README. It looks more professional this way.